src/util.js

```diff
@@ -6,6 +6,8 @@ const multihashes = require('multihashes')
 const multihashing = require('multihashing-async')
 const waterfall = require('async/waterfall')
 
+const BITCOIN_BLOCK_HEADER_SIZE = 80
```

Comment:
Is it standard practice to SHOUT CONSTANTS?

Reply:
Good question, I'm used to shout constants. I couldn't find any pointers to a standard, but I'm used to do that in all the languages I code.